includes/wikia/parser/templatetypes/handlers/TemplateCleanup.class.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+class TemplateCleanup {
+
+	/**
+	 * This method will remove empty sections headers from article content
+	 *
+	 * @param $parser
+	 * @param $html
+	 *
+	 * @return bool
+	 */
+	public static function doCleanup( $parser, &$html ) {
```

Comment:
One thing I noticed about this method's output is that it changes whitespace in lists:
```
<ul><li> 1
</li><li> 2
</li><li> 3
</li></ul>
```
becomes
```
<ul>
<li> 1
</li>
<li> 2
</li>
<li> 3
</li>
</ul>
```
I do not think it's an issue in our case; What's your opinion?